AUTHORS

```diff
@@ -220,6 +220,7 @@ answer newbie questions, and generally made Django that much better:
     Chris Wesseling <Chris.Wesseling@cwi.nl>
     Chris Wilson <chris+github@qwirx.com>
     Ciaran McCormick <ciaran@ciaranmccormick.com>
+    Claire Pritchard <claire.m.pritchard@gmail.com>
```

Comment:
You should make at least a single non-trivial change to add yourself to `AUTHORS`, see [docs](https://docs.djangoproject.com/en/stable/internals/contributing/writing-code/coding-style/#miscellaneous).

Reply:
Oops, I missed that. I was looking at https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#all-tickets. I've taken it out.